index.js

```diff
@@ -83,10 +89,50 @@ Use the finalScore function below to do the following:
 }
 */ 
 
-function finalScore(/*Code Here*/){
-  /*Code Here*/
+function finalScore(inningCB, numInnings){
+  let gameCB = {Home: inningCB(), Away: inningCB(),
+  }
+
+  let homeScore = 0
+  let awayScore = 0
+
+  for(let i=0; i < numInnings; i++){
+    const currentScore = gameCB
+    homeScore = homeScore + currentScore.Home
+    awayScore = awayScore + currentScore.Away
```

Comment:
Awesome job! Another way you could write this is 
`awayScore += currentScore.Away` (if you don't want to write it out each time)